osf/external/gravy_valet/request_helpers.py

```diff
@@ -154,13 +192,159 @@ def _make_gv_request(
             requested_resource=requested_resource,
         )
     )
-    response = requests.get(endpoint_url, headers=auth_headers, params=params)
+    if request_method == 'POST':
+        auth_headers['Content-Type'] = 'application/vnd.api+json'
+        response = requests.post(endpoint_url, headers=auth_headers, json={'data': data})
+    else:
+        response = requests.get(endpoint_url, headers=auth_headers, params=params)
     if not response.ok:
         # log error to Sentry
         pass
     return response
 
 
+def get_citation_url_list(auth, addon_short_name, project, request=None, pid=None):
```

Comment:
I'd rename this method, as it's name isn't clear

Reply:
done